go/tricorder/setup.go

```diff
@@ -116,6 +118,14 @@ func initDefaultMetrics() {
 		resourceUsageGroup,
 		units.None,
 		"Block input operations")
+	if numOpenFileDescriptors >= 0 {
+		RegisterMetricInGroup(
+			"/proc/io/num-open-file-descriptors",
+			&numOpenFileDescriptors,
```

Comment:
Alternatively, you could just pass countOpenFileDescriptors function pointer and keep this metric out of the group. 

Keep in mind that each time you query for this metric, it causes the above RegisterUpdateFunc for the group to be called. Using groups is good if you must use a system call that returns several usage metrics at once. In this case, you don't have that.